packages/fxa-payments-server/src/routes/Product/index.tsx

```diff
@@ -167,22 +173,58 @@ export const Product = ({
 
     // Do we already have a subscription to the product in the selected plan?
     if (customerIsSubscribedToProduct(customerSubscriptions, productPlans)) {
-      return <SubscriptionRedirect {...{ plan: selectedPlan }} />;
+      return (
+        <SubscriptionSuccess
+          {...{
+            plan: selectedPlan,
+            customer: customer.result,
+            profile: profile.result,
+            isMobile,
+          }}
+        />
+      );
     }
   }
 
+  const mobileCreateHeading = isMobile ? (
+    <div className="mobile-subscription-create-heading">
+      <div className="subscription-create-heading">
+        <Localized id="product-plan-details-heading">
+          <h2>Set up your subscription</h2>
+        </Localized>
+        <Localized id="sub-guarantee">
+          <p className="subheading">30-day money-back guarantee</p>
+        </Localized>
+      </div>
+    </div>
+  ) : null;
+
   return (
-    <SubscriptionCreate
-      {...{
-        profile: profile.result,
-        accountActivated,
-        selectedPlan,
-        createSubscriptionAndRefresh,
-        createSubscriptionStatus,
-        resetCreateSubscription,
-        validatorInitialState,
-      }}
-    />
+    <>
+      <Header {...{ profile: profile.result }} />
+      <div className="main-content">
+        <SubscriptionCreate
+          {...{
+            profile: profile.result,
+            accountActivated,
+            selectedPlan,
+            createSubscriptionAndRefresh,
+            createSubscriptionStatus,
+            resetCreateSubscription,
+            validatorInitialState,
+          }}
+        />
+        <PlanDetails
+          {...{
+            profile: profile.result,
+            selectedPlan,
+            isMobile,
+            showExpandButton: isMobile,
```

Comment:
I think this is fine to leave as-is, but I realize seeing this PR in its entirety that you could set `showExpandButton` in `PlanDetails` based off of `isMobile` since they're the same value instead of passing in both values every time we need to render `PlanDetails`.

Reply:
I though about changing that also, but I think it's more clear having them separate, especially in tests and stories

Reply:
We could still leave them separate but set the variable in `PlanDetails` instead of setting it before composing `PlanDetails` every time. Will leave that up to you though.